unconference/volunteer-support-guide.md

```diff
@@ -0,0 +1,118 @@
+management policies and procedures to reduce burnout and improve the experience of their volunteers.     
+
+# Definitions
+*Volunteer* - someone who performs a service or work voluntarily, usually unpaid 
+*Burnout* - emotional or physical exhaustion, usually as a response to chronic emotional and interpersonal stressors on the job
+*Marginalized groups* - groups who historically have been underserved and undervalued, largely due to identity demographics
+(e.g. people who are BIPOC, women, gender minorities, etc.) 
+
+# Problem Statement
+People get involved in volunteering for a number of reasons: as a form of self-education or skill-building, to network and 
+build community, to gain access to open spaces, etc. It is often an entry point into open source and future opportunities. 
+
+The open nature of volunteerism paradoxically can also reproduce societal inequalities. Volunteer works cuts into leisure 
+time and inherently demands stability, resources, energy and time from volunteers – therein lies the source of many issues 
+related to burnout, as volunteering becomes more unsustainable over time. Volunteer participation is dependent on an 
+individual’s available leisure time (and fiscal resources to compensate for that time). This dependency can be especially 
+strenuous for volunteers from marginalized backgrounds, who are historically underserved and may have limited resources, 
+leading many to burnout and dropout over time.
+
+We hope this guide can help organizations be aware of and reduce these barriers to participation for their volunteers. 
+Organizations have a multitude of resources available to them, e.g. monetary funds, personnel, institutional knowledge, 
+which can be leveraged to support volunteers. This guide aims to provide clarity on how organizations can reduce burnout 
+in their volunteers by providing recommendations and inspiration for how organizations can best utilize the resources available to them. 
+
+# Support
+Recognize that volunteering carries an inherent opportunity cost. Every hour spent volunteering is an hour that cannot be spent doing paid work. This opportunity cost disproportionately affects members of marginalized groups especially those who are economically disadvantaged. Organizations should be mindful of this when asking volunteers to donate their time and look for ways to offset this cost.
+
+Depending on the organization’s available resources  
+Organizations with monetary resources can help make volunteering more equitable by paying for items relevant to work-life balance
+Detail of ways to enable economically disadvantaged individuals to take advantage of volunteer opportunities, e.g. travel cost/transportation stipend, covering the cost of child care, internet, covering accommodations, per diems 
+Challenges to monetary compensation
+Tax implications - make sure you’re aware of the laws regarding payments to individuals
```

Comment:
note that this can involve the home country of the project as well as the country of the volunteer